app/assets/stylesheets/workarea/storefront/content_search/components/_content_search_autocomplete.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Extensions to the search autocomplete plugin to support content
+ * results.
+ */
+
+.search-autocomplete__content {
```

Comment:
The file should be named `_search_autocomplete.scss` in this case... Or you can change this selector to be `.content-search-autocomplete` but the filename and CSS block name should be congruent.